platform/machine/unprivqemu/cluster.go

```diff
@@ -94,6 +94,7 @@ func (qc *Cluster) NewMachineWithOptions(userdata *conf.UserData, options platfo
 	builder := platform.NewBuilder(board, confPath)
 	defer builder.Close()
 	builder.Uuid = qm.id
+	builder.Firmware = qc.flight.opts.Firmware
```

Comment:
Unrelated to this PR but are we missing the setting of `builder.Swtpm`? It's being defaulted to `true` in `platform.NewBuilder` and a cursory scan didn't see it being set to the value of `qc.flight.opts.Swtpm` anywhere.

Reply:
I think we should nuke that option and just set it off if !x86_64 - will do it in a followup PR.